src/components/pathway-progress/tests/PathwayProgressListingCard.test.jsx

```diff
@@ -19,22 +18,21 @@ jest.mock('react-router-dom', () => ({
   useLocation: jest.fn(),
 }));
 
-const PathwayProgressListingCardWithContext = ({ initialAppState, initialUserSubsidyState, pathwayData }) => (
+jest.mock('../../app/data', () => ({
+  ...jest.requireActual('../../app/data'),
+  useEnterpriseCustomer: jest.fn().mockReturnValue({
+    data: { slug: 'test-enterprise-slug' },
```

Comment:
[nit] opportunity for rosie factory with enterpriseCustomer, FWIW not hugely opinionated here though 👍🏽